### ISSUE TITLE
Add support for @storybook/builder-vite

### ISSUE THREAD

**Issue by Tscheffrey** [philip-hartmann/storybook-typo3fluid#1]
Hey, thanks for this great tool!   
As far as I can tell, this module also works really well with [@storybook/builder-vite](https://www.npmjs.com/package/@storybook/builder-vite). There is one catch: [Vite environment variables](https://vitejs.dev/guide/env-and-mode.html) work a bit differently.

- They are accessed by using `import.meta.env.*`
- They need to have the `VITE_` prefix to be accessible on the client side

My quick fix was to replace variables like `process.env.TYPO3FLUID_API_URL` with  `import.meta.env.VITE_TYPO3FLUID_API_URL` but this will make the module unusable with the original webpack builder.

Possible solutions:

1. Add an expression that checks for the `process.env.*` type env variables and if they are not present, use the `import.meta.env.VITE_`type variables. (preferred)
2. Add a parameter for the `FluidTemplate` class that tells it which builder is being used. However, this would have to be set in every story.

**Comment by philip-hartmann** [comment]
Hello @Tscheffrey, thanks for checking out the project!

Your first proposed solution should be the right one. It's also an easy fix. I just have to configure babel too because the nullish coalescing operator throws an error in storybook.

**Comment by IanVS** [comment]
Note that it's also possible to configure which prefixes are allowed, via `envPrefix`: https://vitejs.dev/config/shared-options.html#envprefix

**Comment by philip-hartmann** [comment]
@IanVS Thanks for the link! Multiple prefixes are really helpful.

@Tscheffrey Webpack 4 & 5, the default builder for storybook, do not support import.meta in the currently used version from storybook. This might change with storybook 7.0. Until then we have to use an additional entrypoint `storybook-typo3fluid/dist/vite` to support `@storybook/builder-vite`. I've bumped the package version to 0.2.0. You also have to update the environment variable names and some Vite related changes I have documented in the readme section [builder differences](https://github.com/philip-hartmann/storybook-typo3fluid#builder-differences).
